src/util/constants.ts

```diff
@@ -4,4 +4,5 @@ export const BASE_API_PREFIX = '/v1/api/';
 export const CRICINGIF_URL = {
     BASE_URL: 'https://webapi.cricingif.com/api',
     FULL_SCORE_CARD: '/matches/{matchId}/fullscorecard'
-}
\ No newline at end of file
+}
+export const CURRENT_LEAGUE_ID = 1
```

Comment:
This should be 2 because we are dealing with PSL 2020 currently.

Reply:
The migration data was for league_id 1, but we won't be needing this anymore since @mideveloper mentioned to move the static ID to FE side